tools/dev/setup-toolchain.sh

```diff
@@ -21,56 +21,58 @@
 
 # Set working directory.
 export CURDIR=`pwd`
-export WORKDIR=$CURDIR/nanvix-toolchain
-mkdir -p $WORKDIR
+export WORKDIR=$CURDIR/toolchain/i386
```

Comment:
@Theldus  shouldn't it be export WORKDIR=$CURDIR/tools/dev/toolchain/i386

Reply:
@ppenna Okay, I'll change. I thought the script would be invoked from within the 'dev' folder.